frontend/src/scenes/pipeline/hogfunctions/hogFunctionTestLogic.tsx

```diff
@@ -66,11 +69,23 @@ export const hogFunctionTestLogic = kea<hogFunctionTestLogicType>([
                 setTestResult: (_, { result }) => result,
             },
         ],
+
+        savedGlobals: [
+            [] as { name: string; globals: HogFunctionInvocationGlobals }[],
+            { persist: true, prefix: `${getCurrentTeamId()}__` },
```

Comment:
Personally I think this is fine. I'd say, lets add some tracking to this. If we find a lot of usage then we can consider a more robust persistence?

Reply:
I added a bunch of `data-attr`-s all over the place and called these things "test data" everywhere. I guess good enough?